projects/kit/components/slider/slider.component.ts

```diff
@@ -57,15 +74,9 @@ export class TuiSliderComponent {
     }
 
     public get valueRatio(): number {
-        return (this.value - this.min) / (this.max - this.min) || 0;
-    }
-
-    public get min(): number {
-        return Number(this.el.min);
-    }
+        const range = this.max() - this.min();
 
-    public get max(): number {
-        return Number(this.el.max || 100);
+        return (range && (this.value - this.min()) / range) || 0;
```

Comment:
Why change that part? Seemed easier to read as a one-liner to me :)

Reply:
Native slider behaves in the such way

https://github.com/taiga-family/taiga-ui/blob/55b4fb84b65ce8194ffcd00fa91e66285aff62b3/projects/kit/components/slider/test/slider.spec.ts#L176-L189

- https://github.com/taiga-family/taiga-ui/pull/1703

Reply:
No, I mean why moving `range` in a separate const? Why not:
```ts
return (this.value - this.min()) / (this.max() - this.min()) || 0;
```

Reply:
It is not interchangeable code.
If  `this.max() - this.min()` return `0` => all division returns `Infinity`.